diff --git a/node_modules/@samverschueren/stream-to-observable/index.js b/node_modules/@samverschueren/stream-to-observable/index.js
index 962afcc..77f6941 100644
--- a/node_modules/@samverschueren/stream-to-observable/index.js
+++ b/node_modules/@samverschueren/stream-to-observable/index.js
@@ -1,5 +1,5 @@
 'use strict';
-const Observable = require('any-observable');
+const Observable = require('zen-observable');
 
 function or(option, alternate, required) {
 	const result = option === false ? false : option || alternate;
